README.md

```diff
@@ -91,3 +91,21 @@ https://metasource.gridhead.net/
     ```
 12. Consider contributing to the project with methods that you see feasible.
 
+### Using Docker
+
+1.  Clone the repository contents to your local projects directory.
```

Comment:
Please start with providing instructions to install `podman`.

```
$ sudo dnf install podman
```

Then you can proceed with the instructions that you have mentioned.